grouper/fe/templates/email/nonauditor_html.tmpl

```diff
@@ -0,0 +1,11 @@
+{% extends "email/base_html.tmpl" %}
+
+{% block subject %}Membership in '{{ group_name }}' set to expire{% endblock %}
+
+{% block content %}
+<p>{{ group_name }} is an audited group, but you are not an auditor. Because of this, your
+membership in {{ group_name }} has been set to expire. You may resolve this by becoming an
+auditor and then having an owner or np-owner in {{ group_name }} reset your expiration time.
```

Comment:
We should provide some guidance about how you become an auditor. I think there's an existing setting for this?